src/Backend/Modules/Locale/Layout/Templates/Index.html.twig

```diff
@@ -66,14 +66,14 @@
               <div class="form-group">
                 <label for="name" class="control-label">
                   {{ 'lbl.ReferenceCode'|trans|ucfirst }}
-                  <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpName'|trans }}"></abbr>
+                  <abbr class="fa fa-info-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.HelpName'|trans }}"></abbr>
                 </label>
                 {% form_field name %} {% form_field_error name %}
               </div>
               <div class="form-group">
                 <label for="value" class="control-label">
                   {{ 'lbl.Value'|trans|ucfirst }}
-                  <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpValue'|trans }}"></abbr>
+                  <abbr class="fa fa-info-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.HelpValue'|trans }}"></abbr>
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320